cmd.go

```diff
@@ -133,6 +133,14 @@ func (ctx *Context) Infof(format string, params ...interface{}) {
 	}
 }
 
+// Warningf allows for the logging of messages, at the warning level, from a
+// command's context. This is useful for logging errors which do not cause a
+// command to fail (e.g. an error message used as a deprecation warning that
+// will be upgraded to a real error message at some point in the future.)
+func (ctx *Context) Warningf(format string, params ...interface{}) {
+	logger.Logf(loggo.WARNING, format, params)
```

Comment:
Probably worthwhile adding a comment to explain why Logf is used rather than Warningf.

perhaps something like '// Use Logf so loggo captures the caller of the Context.Warningf as the log source.'

Also, can you change Infof and Verbosef on Context to use:

```go
logger.Logf(loggo.INFO, format, params)
```
rather that `logger.Infof`